README.md

```diff
@@ -52,6 +52,17 @@ If I wanted to make sure that mintapi used the chromium executable in my /usr/bi
 ```
 where prepending the /usr/bin path to path will make those binaries found first. This will only affect the cron job and will not change the environment for any other process.
 
+#### Docker Image
+You can also use the docker image to help manage your environment so you don't have to worry about chrome or chromedriver versions. There are a few caveats:
+1. Headless mode is recommended. GUI works but introduces the need to configure an X11 server which varies with setup. Google is your friend.
+2. Almost always use the flag `--use-chromedriver-on-path` as the chrome and chromedriver built into the docker image already match and getting the latest will break the image.
+3. If you want to persist credentials or your chrome session, you'll need to do some volume mounting.
```

Comment:
Any issues with using the config file?

Reply:
You'll have to specify the path from the perspective of the container and it either needs to be inside the container or mounted to it; but other than that, no.